hunts/src/puzzle-table.js

```diff
@@ -26,7 +26,15 @@ const TABLE_COLUMNS = [
     Header: "Name",
     accessor: "name",
     Cell: NameCell,
-    className: "col-4",
+    className: "col-3",
+  },
+  {
+    Header: "Tags",
+    id: "tags",
+    Cell: TagCell,
+    className: "col-3",
+    accessor: (row) => row.tags.map(({ name }) => name).join(" "),
```

Comment:
definitely not appropriate for this commit, but the matching algorithm is pretty overeager because its picking up this string mostly